### PR TITLE
fix(FOROME-358): drawer blocks layout

### PR DESCRIPTION
**Pull request by Lapk1n**
Changing the order of drawer blocks is being saved after opening.

### CODE REVIEW

File: src/components/variant/drawer.tsx [hunk 1]
```diff
@@ -20,9 +21,11 @@ export const VariantDrawer = observer(
   (): ReactElement => {
     const drawerWidth = window.innerWidth - 380
     const localStorageData = window.sessionStorage.getItem('gridLayout')
```
Comment (AndreyUstyumenko) [inline src/components/variant/drawer.tsx:23]: `SessionStoreManager.read< IGridLayout >('gridLayout')`

File: src/components/variant/drawer.tsx [hunk 2]
```diff
@@ -20,9 +21,11 @@ export const VariantDrawer = observer(
...
-    const currentGridLayout = localStorageData && JSON.parse(localStorageData)
 
-    const [layout, setLayout] = useState(
+    const currentGridLayout: IGridLayout[] =
+      localStorageData && JSON.parse(localStorageData)
```
Comment (AndreyUstyumenko) [inline src/components/variant/drawer.tsx:26]: delete

File: src/components/variant/ui/body.tsx [hunk 3]
```diff
@@ -166,10 +176,10 @@ export const VariantBody = observer(
             variantStore.checkRecodsDisplaying()
             window.sessionStorage.setItem(
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/body.tsx:177]: `SessionStoreManager.write('gridLayout', sortGridLayout(layoutData))`

File: src/components/variant/ui/body.tsx [hunk 4]
```diff
@@ -166,10 +176,10 @@ export const VariantBody = observer(
...
               'gridLayout',
-              JSON.stringify(layoutData),
+              JSON.stringify(sortGridLayout(layoutData)),
             )
 
-            setLayout(layoutData)
+            setLayout(sortGridLayout(layoutData))
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/body.tsx:182]: move `sortGridLayout(layoutData)` to a const

File: src/components/variant/ui/body.tsx [hunk 5]
```diff
@@ -129,6 +135,10 @@ export const VariantBody = observer(
       indicator = 1
     })
 
+    const sortGridLayout = (gridLayout: IGridLayout[]) => {
+      return gridLayout.sort((a, b) => (a.y > b.y ? 1 : -1))
```
Comment (AndreyUstyumenko) [inline src/components/variant/ui/body.tsx:139]: looks like `(a, b) => (a.y - b.y)` would work

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by YaroslavTrefilovBraveDevelopers — APPROVED** [review]

**Pull request merged**